src/lib/index.js

```diff
@@ -449,12 +449,12 @@ lib.addStyleRule = function(selector, styleString) {
 
 lib.getTranslate = function(element) {
 
-    var re = /(\btranslate\()(\d*\.?\d*)([^\d]*)(\d*\.?\d*)([^\d]*)(.*)/,
+    var re = /.*\btranslate\((\d*\.?\d*)[^\d]*(\d*\.?\d*)[^\d].*/,
```

Comment:
The regex is being used in `String.replace` which only substitutes the matched pattern - this just captures _everything_ in the string and allows replacement with only the two sections (the number values) it needs. I forgot to add a test for the case that it missed, but I'll add it now.